R/fit.R

```diff
@@ -1,9 +1,10 @@
 #' Fit epidemiological delay distributions using a `brms` interface
 #'
-#' @inheritParams epidist_validate
-#' @param formula A formula object created using `brms::bf`. A formula must be
-#' provided for the distributional parameter `mu` common to all `brms` families.
-#' Optionally, formulas may also be provided for additional distributional
+#' @param data A `data.frame` containing line list data.
+#' @param formula An object of class [stats::formula] or [brms::brmsformula]
```

Comment:
This seems like bleed through from #396

Reply:
No it's just bad restraint from me. I wanted to update the family documentation then it was just easy to update the documentation for other arguments of `epidist`

Reply:
Can you flag this in #396 as having happened here. Ideally avoid doing this in the future.